### ISSUE TITLE
Command code redundant?

### ISSUE THREAD

**Issue by elthran** [elthran/RPG-Game#211]
In lines 826+ of app.py you have command code but now you also have it in commands.py

Are these redundant or do they do different things? Can you please explain it to me? :)

**Comment by klondikemarlen** [comment]
This:
```
try:
        response = Command.cmd_functions(cmd)(myHero, database=database, arg_dict=request.args)
        database.update()
        # pdb.set_trace()
        return response
    except AttributeError:
        print("Warning: Using old code for command: '{}'".format(cmd))
# Look in the not yet refractored list of if statemens ...
```

in app.py runs commands.py Command class.

Everything after that is the old code. Which may or may not be still useful. I was hoping to transition everything after that into commands.py at some point.

The lines above:
```
if cmd == 'favicon.ico':
        return "success", 200, {'Content-Type': 'text/plain'}

    testing = False # True
    if testing:
        print('request is:', repr(request))
        # print('request data:', repr(request.data))
        # print("request form:", repr(request.form))
        print('request view_args:', repr(request.view_args))
        print('request args:', repr(request.args))
        print('cmd is:', repr(cmd))

    # event = Event(request.args)
    # event.add["hero"] = myHero
# event.add["database"] = database
```
are for testing and debugging. Which I end up doing a lot of.

Lastly:
```
@app.route('/<cmd>') # need to make sure this doesn't conflict with other routes
def command(cmd=None):
```
Is the most critical as it is what is called by having a `<button class="command">`.
Well the JS/AJAX code sends data in through this _url_.

**Comment by klondikemarlen** [comment]
I have fixed some of this in f0d3af1188f941b376a1439ef5ec938fb6c7be5c branch https://github.com/elthran/RPG-Game/tree/issue133_prevent_url.

**Comment by klondikemarlen** [comment]
I think that all of the old command code is now gone.
